src/displayapp/screens/Screen.h

```diff
@@ -12,13 +12,25 @@ namespace Pinetime {
           Screen(DisplayApp* app) : app{app} {}
           virtual ~Screen() = default;
 
-          // Return false if the app can be closed, true if it must continue to run
+          /**
+           * Most of the time, apps only react to events (touch events, for example).
+           * In this case you don't need to do anything in this method.
+           *
+           * For example, InfiniPaint does nothing in Refresh().
+           * But, if you want to update your display periodically, draw an animation...
+           * you cannot do it in a touch event handler because these handlers are not
+           * called if the user does not touch the screen.
+           *
+           * That's why Refresh() is there: update the display periodically.
+           *
+           * @return false if the app can be closed, true if it must continue to run
+           **/
           virtual bool Refresh() = 0;
 
-          // Return false if the button hasn't been handled by the app, true if it has been handled
+          /** @return false if the button hasn't been handled by the app, true if it has been handled */
```

Comment:
Is this notation needed for a specific documentation processor?

Reply:
It's the Doxygen/Javadoc format that things like CLion can parse. If you press Ctrl+Q while the cursor is on a documented variable, it'll show that documentation.

Reply:
That's really useful, thanks!